csharp/test/Drivers/Apache/ApacheTestConfiguration.cs

```diff
@@ -51,5 +51,8 @@ public class ApacheTestConfiguration : TestConfiguration
         [JsonPropertyName("polltime_milliseconds"), JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
         public string PollTimeMilliseconds { get; set; } = string.Empty;
 
+        [JsonPropertyName("request_timeout_ms"), JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
```

Comment:
Same here -- for the JSON property and C# property name.

Reply:
Updated properties and options to indicate its use with the HTTP type connection.